data/preset/library-list.yaml

```diff
@@ -2,6 +2,13 @@
 vars:
   app-banner:
     shortdef: 'Banner.Dynamic(mode: full)'
+  app-brand:
+    type: Decoration.ThemeableImage
+    properties:
+      halign: end
+      valign: end
+      expand: true
+      image-uri: 'resource:///app/assets/watermark'
```

Comment:
Is there any way to make the align/expand properties of this configurable but not the `image-uri`? Otherwise, we effectively have two ways to customize it: including an asset at `resource:///app/assets/watermark` or not, and overriding this variable or not.

Reply:
You might want to use something completely different here, like a dynamic logo. I think the fact that we can also customize this default by providing a different watermark is just circumstantial.

Reply:
Looking for an asset at `resource:///app/assets/watermark` was intentional, just like looking for one at `resource:///app/assets/logo` for the dynamic logo.

Reply:
I'm not sure I agree with the point about putting something completely different here. Sure, you could put a ContentGroup, but I'm not super excited about allowing that.